HW6/smonets/3.py

```diff
@@ -0,0 +1,10 @@
+def num_char(my_str):
```

Comment:
another way:
def count_symbols_in_row(word):
    result = {}
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result